### PR TITLE
Event callbacks fixes

### PR DESCRIPTION
**Pull request by reyaleman**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
Fixed some event callbacks that were missing when they got updated (#4395)

**Issues addressed:** 

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Review by EPuncker — APPROVED** [review]

**Review by EvilHero90 — APPROVED** [review]
looks good to me

File: data/scripts/events/player/default_onUpdateStorage.lua [hunk 1]
```diff
@@ -7,7 +7,7 @@ ec.onUpdateStorage = function(creature, key, value, oldValue, isSpawn)
 		return
 	end
 
-	local player = Player(creature)
+	local player = Player(creature:getId())
```
Comment (MillhioreBT) [inline data/scripts/events/player/default_onUpdateStorage.lua:10]: totally unnecessary.

**Review by MillhioreBT — CHANGES_REQUESTED** [review]

File: data/scripts/events/player/default_onUpdateStorage.lua (hunk 1 above)
Comment (reyaleman) [reply data/scripts/events/player/default_onUpdateStorage.lua:10]: I've triple checked and it doesn't work by passing the creature userdata to the player constructor... 

Can you test it by yourself, please?

Comment (MillhioreBT) [reply data/scripts/events/player/default_onUpdateStorage.lua:10]: I think you're right, I tested it on my custom engine and it worked, however it didn't work on TFS master, sorry for the inconvenience, I'll approve it.

**Review by MillhioreBT — APPROVED** [review]

**Pull request merged**